Resources/Prototypes/Entities/Objects/Fun/immovable_rod.yml

```
@@ -66,13 +66,14 @@
   suffix: Wizard
   components:
   - type: ImmovableRod
-    minSpeed: 35
+    minSpeed: 25
+    maxSpeed: 25
     destroyTiles: false
     randomizeVelocity: false
```

```suggestion
    maxSpeed: 25
    destroyTiles: false
    randomizeVelocity: true
```
I’d recommend setting `maxSpeed` to 25 and removing `minSpeed`, so it falls back on the default 10f. That way, speed isn’t fixed. Also, consider setting `randomizeVelocity` to true so speed can vary instead of staying at a constant value.

---

```suggestion
    maxSpeed: 25
    destroyTiles: false
    randomizeVelocity: true
```
Another option is to set `randomizeVelocity` to true and remove `minSpeed` (or tweak it if you don’t want a 10f–25f range), so the rod’s speed varies instead of staying fixed.